### ISSUE TITLE
may i have a look at the training loss curve

### ISSUE THREAD

**Issue by bokesyo** [tloen/alpaca-lora#381]
Dear author:
Thank you for your great work, i have used your code to train my model, but this is my first time to finetune a causal language model, i have no idea about what is a good loss curve, so i would like to have a look at yours, could you please help me, thank you a lot!

**Comment by Oxi84** [comment]
I can describe mine, the train loss goes down from 2.5 to 0.1 in epoch 10, while validation loss goes down to 1.15 at epoch 5, and after it goes up.

**Comment by lywinged** [comment]
> I can describe mine, the train loss goes down from 2.5 to 0.1 in epoch 10, while validation loss goes down to 1.15 at epoch 5, and after it goes up.

You should set early stopping to monitor the loss, but it seems like the invalid training caused by other issues. What are your based model and train_data?

**Comment by bokesyo** [comment]
@Oxi84 Thanks! That is cool, mine is still 0.8 after 1k steps, my batch size=128, lora-r=8, base_model=llama-7b, i used int8 training. :D
